src/connect.js

```diff
@@ -19,8 +19,11 @@
  * @param {Object} baseParams
  * @return {Function}
  */
+
 function connect(params) {
-  // write code here
+  return function(extraParams) {
```

Comment:
es6 arrow function

Reply:
Ya, man! thanks! fixed this. You're working extra time, I really appreciate this! have a good weekend, and remember that's your time.